simulations/geometryXYVxVy/landau/landau4d_fft.cpp

```diff
@@ -90,10 +90,10 @@ int main(int argc, char** argv)
 
     IdxRangeVxVyXY idxrange_vxvyxy_v2Dsplit(idxrange_spvxvyxy_v2Dsplit);
     IdxRangeXYVxVy idxrange_xyvxvy_x2Dsplit(idxrange_spxyvxvy_x2Dsplit);
-    SplineXBuilder const builder_x(idxrange_vxvyxy_v2Dsplit);
-    SplineYBuilder const builder_y(idxrange_vxvyxy_v2Dsplit);
-    SplineVxBuilder const builder_vx(idxrange_xyvxvy_x2Dsplit);
-    SplineVyBuilder const builder_vy(idxrange_xyvxvy_x2Dsplit);
+    SplineXBuilder const builder_x(idxrange_x);
+    SplineYBuilder const builder_y(idxrange_y);
+    SplineVxBuilder const builder_vx(idxrange_vx);
+    SplineVyBuilder const builder_vy(idxrange_vy);
```

Comment:
@tpadioleau It seems that this change is not compulsory. Is this expected?

Reply:
Yes, we thought we should not break too much code at once :D but we were definitely thinking of removing this constructor in the future. We will mark it as deprecated soon.